src/registrar/tests/test_models.py

```diff
@@ -0,0 +1,48 @@
+from django.test import TestCase
+from django.db.utils import IntegrityError
+
+from registrar.models import Contact, DomainApplication, User, Website
+
+
+class TestDomainApplication(TestCase):
+    def test_empty_create_fails(self):
+        """Can't create a completely empty domain application."""
+        with self.assertRaisesRegex(IntegrityError, "creator"):
+            DomainApplication.objects.create()
```

Comment:
I wouldn't ask you to exhaustively test the FSM, but it'd be nice to have at least one test case to minimally prove that you can't, say, create an application in the `APPROVED ` status.

Reply:
Added a couple test cases for the `submit` method.